pkg/service/deploy.go

```diff
@@ -73,6 +74,36 @@ func (s *DeployService) Deploy(ctx context.Context, overrides *DeployOverrides)
 	return nil
 }
 
+// PostDeploy is intended to run after a successful call to Deploy()
+func (s *DeployService) PostDeploy(ctx context.Context) error {
+
+	if ctx.Value(constants.DeployLogFile) == nil {
+		deployLogFileName := s.Util.GetLogFile()
+		ctx = context.WithValue(ctx, constants.DeployLogFile, deployLogFileName)
+	}
+
+	apiURL, err := s.Util.GetOutput(ctx, "api_url")
+
+	if err == nil {
+		url, err := url.Parse(apiURL)
+		if err == nil {
+			hostName := url.Hostname()
+			basePath := url.EscapedPath()
+			s.Config.API.Host = &hostName
+			s.Config.API.BasePath = &basePath
```

Comment:
awesome, way to sneak that feature in there 😉

Reply:
👍 Who, me?? 😉